clang/Token.d

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright: Copyright (c) 2016 Wojciech Szęszoł. All rights reserved.
+ * Authors: Wojciech Szęszoł
+ * Version: Initial created: Feb 14, 2016
+ * License: $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0)
+ */
+ 
+module clang.Token;
+
+import std.typecons;
+import std.conv : to;
+import clang.c.Index;
```

Comment:
Please group the imports. First standard library, then third party libraries, last current project. Separate with newline.